application/utils/auth.py

```diff
@@ -11,6 +11,7 @@ def generate_token(user, expiration=TWO_WEEKS):
     s = Serializer(app.config['SECRET_KEY'], expires_in=expiration)
     token = s.dumps({
         'id': user.id,
+        'profile': user.name,
```

Comment:
Is a profile name required when you sign up? Otherwise we may get null reference errors

Reply:
This method will only work for new Registered users after the PR is merged.
If you use existing users (before PR is merged), will Profile will display Null